mta/entities/global_configuration.py

```diff
@@ -152,33 +153,170 @@ def is_displayed(self):
         )
 
 
+class SystemLabelsView(BaseLoggedInPage):
+    """This view represents System rules tab page"""
+
+    paginator = Pagination(locator='.//div[contains(@class, "pf-c-pagination")]')
+
+    table = Table(
+        locator='.//table[contains(@aria-label, "Table")]',
+        column_widgets={
+            "Provider ID": Button(locator='.//th[@data-label="Provider ID"]/button'),
+            2: Dropdown(),
+        },
+    )
+    search = Input(locator='.//input[@aria-label="Filter by provider ID"]')
+
+    def search_project(self, provider_id):
```

Comment:
@ganeshhubale : We should not be adding methods like search , delete etc to View classes . A collection is needed when we can create more than one object of anything like in Projects , we can create more than one project . SO we can have a collection for projects. 

In system rules we are not creating anything just validating things so we don't need Global collection . 

You have register your navigations like 
      https://github.com/windup/windup_integration_test/blob/master/mta/entities/analysis_results.py#L147

Reply:
I have followed `AnalysisResults` way and improved automation.